__tests__/integration/schema-version.test.ts

```diff
@@ -0,0 +1,19 @@
+import { fql } from "../../src";
+import { getClient } from "../client";
+
+describe("schema version is returned by the client", () => {
+  const client = getClient();
+  it("returns the schema version", async () => {
+    const resTs = await client.query(fql`
+    Collection.create({ name: "TestColl" })
+    `);
```

Comment:
looks like the rest of their tests indent these:
```suggestion
    const resTs = await client.query(fql`
      Collection.create({ name: "TestColl" })
    `);
```